### DISCUSSION TITLE
Need to Implement Slick Carousel in Breeze theme

### DISCUSSION THREAD

**Discussion by JamesDenish** [breezefront/community#184] [1. Help] [answered]
I have implemented slick into breeze theme but it's not working and I have used navigation pro extension for megamenu and want category in slider form so can you please help me out how to get category into slider form.

Kindly help me out it's urgent need for me and want to implement as soon as possible. 

I'm getting error 
Uncaught TypeError: $(...).slick is not a function at index.js file using breeze theme.

**Comment by vovayatsyuk** [comment]
I recommend using the built-in slider: https://breezefront.com/docs/slider

I don't fully understand the part about navigation and what are you trying to do with it.

**Reply by JamesDenish** [reply]
I want categories which are displayed in menu want to display in slider form

![footer](https://github.com/breezefront/community/assets/142228975/1d5add57-dd62-427f-9bf6-d5d72f2e86e4)
I want this subcategory which are in bold want to display it in slider
For example 
Konstruktive Schutzverpackungen|Verpackung nach Mass|Bedruckte Verpackungen|Folien Haubenservice

**Reply by vovayatsyuk** [reply]
Try this very basic implementation:

```js
$('ul.children li').css('flex-shrink', 0);
$('ul.children').addClass('slick-list').css('flex-wrap', 'nowrap');
$('ul.children').parent().css('position', 'relative').pagebuilderSlider();
```

**Reply by JamesDenish** [reply]
Should I used it like as below:-
require(['jquery'], function($){
		$(document).on('breeze:load', function () {
    		$(document).ready(function(){
    		$('ul.children li').css('flex-shrink', 0);
$('ul.children').addClass('slick-list').css('flex-wrap', 'nowrap');
$('ul.children').parent().css('position', 'relative').pagebuilderSlider();
    		})
    	});
   });
   Please do let me know.

**Reply by vovayatsyuk** [reply]
https://breezefront.com/docs/custom-javascript

**Reply by JamesDenish** [reply]
Do I need any packages for using pagebuilderSlider() into breeze theme for magento 2 ?

As of now it's showing me error in console that pagebuilderSlider() is not a function.

**Reply by vovayatsyuk** [reply]
Can you share the link to your site?

**Reply by JamesDenish** [reply]
Right now I am working in local. Should I provide you the screenshot for following error ?

**Reply by vovayatsyuk** [reply, marked as answer]
It's possible that the js bundle with slider is not added to the page if you don't have the Magento_Pagebuilder module. Try to create the following `breeze_default.xml` layout update in your theme:

```xml
<?xml version="1.0"?>
<page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
    <body>
        <referenceBlock name="breeze.js">
            <arguments>
                <argument name="bundles" xsi:type="array">
                    <item name="cms" xsi:type="array">
                        <item name="active" xsi:type="boolean">true</item>
                    </item>
                </argument>
            </arguments>
        </referenceBlock>
    </body>
</page>
```

**Reply by JamesDenish** [reply]
Thanks @vovayatsyuk for helping me out you  saved my deadline.

It worked just my adding this reference block in my default_head_blocks.xml
